Controller/TemplatingExceptionController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Bundle\FrameworkBundle\Templating\TemplateReference;
+use Symfony\Component\HttpFoundation\Request;
+
+abstract class TemplatingExceptionController extends ExceptionController
+{
+    protected $templating;
+
+    public function __construct(
+        ExceptionWrapperHandlerInterface $exceptionWrapperHandler,
+        FormatNegotiator $formatNegotiator,
+        ViewHandlerInterface $viewHandler,
+        array $exceptionCodes,
+        array $exceptionMessages,
+        $showException,
+        EngineInterface $templating
+    ) {
+        parent::__construct($exceptionWrapperHandler, $formatNegotiator, $viewHandler, $exceptionCodes, $exceptionMessages, $showException);
```

Comment:
@lsmith77 this does not change the fact that your factory method will need to receive all deps, otherwise you have the same issue about creating an incomplete object.